tests/testsuite/alt_registry.rs

```diff
@@ -620,7 +620,7 @@ Caused by:
         .run();
 
     for cmd in &[
-        "init", "install", "login", "owner", "publish", "search", "yank",
+        "init", "install foo", "login", "owner", "publish", "search", "yank",
```

Comment:
was this intended?

Reply:
Yes, I should have left a comment.  `install --registry` now requires a package name (`foo`) and that validation is done before the `--registry` name validation.